lib_cxng/include/lcx_ripemd160.h

```diff
@@ -71,7 +71,10 @@ cx_err_t cx_ripemd160_init_no_throw(cx_ripemd160_t *hash);
  */
 static inline int cx_ripemd160_init(cx_ripemd160_t *hash)
```

Comment:
This one should be `DEPRECATED` no?

Reply:
You're certainly right, but its like that on master and its not the topic of this PR, so I leave this for some volunteer ;-)
(and I just checked, it is not called today - but its not a reason to not fix it and check for other prototypes in the same case)